doc/manual/packages/gcs-substituter.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<section xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         version="5.0"
+         xml:id="ssec-gcs-substituter">
+
+  <title>Serving a Nix store via Google Cloud Storage</title>
+
+  <para>Nix has built-in support for storing and fetching store paths
+  	from Google Cloud Storage.</para>
+  <para>For a GCS bucket with the name <literal>example-nix-cache</literal> the URI is <uri>gs://example-nix-cache</uri>.</para>
+  <para>Nix will use the environment variable <literal>GOOGLE_APPLICATION_CREDENTIALS</literal> to discover the credentials JSON file.</para>
```

Comment:
We should probably write we do authentication as documented at https://cloud.google.com/storage/docs/authentication#libauth - which also includes automatic credential management in Google App Engine and Google Cloud engine via their metadata server.

`GOOGLE_APPLICATION_CREDENTIALS` is another way to do it, but this reads as if it's the only way to do it. I'd still keep the example with `GOOGLE_APPLICATION_CREDENTIALS` however.